### DISCUSSION TITLE
SEN0343 or two of SparkFun SEN-16476 or Adafruit 3965

### DISCUSSION THREAD

**Discussion by skupu** [arendst/Tasmota#19352] [Ideas]
Dear Theo @arendst,

I need to measure pressure difference - the most suitable way would be with one DFRobot SEN0343 or two of SparkFun SEN-16476 or Adafruit 3965.

Is there any chance to make use of those with Tasmota?

There are libraries:
https://github.com/DFRobot/DFRobot_LWLP
https://github.com/sparkfun/SparkFun_MicroPressure_Arduino_Library
https://github.com/adafruit/Adafruit_MPRLS

but honestly I don't know how to integrate them into own compiled tasmota, if possible.

Could you please support?

Thank you!

_Originally posted by @skupu in https://github.com/arendst/Tasmota/discussions/19300_

**Comment by barbudor** [comment]
Posting twice the same request won't help
At worst, you will piss off the team and get ghosted

To create a driver start reading this : https://tasmota.github.io/docs/For-Developers/
Then try to identify the most similar existing driver and get inspiration from it

**Comment by skupu** [comment]
@barbudor - thank you - that is at least a reaction.
Thanks for suggesting.
I will try to investigate but I'm not sure if I am advanced enough to succeed.
I was hoping someone had similar issue and will give me a hand.
Still counting on the more experienced members of the community.
